cmd/vsphere-xcopy-volume-populator/internal/populator/secure-vmkfstools-wrapper.py

```
@@ -0,0 +1,311 @@
+            "taskId": task_id,
+            "pid": pid,
+            "exitCode": exit_code,
+            "lastLine": last_line,
+            "stdErr": stderr_content,
+            "operation": "status"
+        }
+        
+        logging.info(f"Status check for task {task_id}: {result}")
+        print(XML_TEMPLATE.format(status="success", message=json.dumps(result)))
+        
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)